packages/frontend/happy-frontend.cabal

```diff
@@ -38,7 +38,9 @@ tested-with:
         GHC == 7.0.4
 
 flag bootstrap
-  description: Optimize the implementation of happy using a pre-built happy
+  description:
+    Optimize the implementation of happy using a pre-built happy,
+    and add support for attribute grammars.
```

Comment:
Should mention the other behavioral thing this flag does.

Reply:
I would have the flag the other way round, so it would then be "Build a version of happy for boot-strapping.  The generated binary will not have support for attribute grammars."
See https://github.com/haskell/happy/pull/256#discussion_r1342398966.

Reply:
It's going to be confusing either way because `bootstrap` can be interpreted either as "bootstrapped" or "for bootstrapping", which are opposite in meaning.

Any ideas for a less ambiguous name?

Reply:
Well that is a preexisting issue; I am happy to change it but seems like ti should be done separately. Does all the new things from this PR look good?